languages/python/atom/element.py

```
@@ -334,21 +334,22 @@ def get_all_commands(self, element_name=None, ignore_caller=True):
                 command_list.extend([f'{element}:{command}' for command in elem_commands])
         return command_list
 
-    def command_add(self, name, handler, timeout=RESPONSE_TIMEOUT, deserialize=False):
+    def command_add(self, name, handler, timeout=RESPONSE_TIMEOUT, deserialize=False, serialization="msgpack"):
         """
         Adds a command to the element for another element to call.
 
         Args:
             name (str): Name of the command.
             handler (callable): Function to call given the command name.
             timeout (int, optional): Time for the caller to wait for the command to finish.
-            deserialize (bool, optional): Whether or not to deserialize the data using msgpack before passing it to the handler.
+            deserialize (bool, optional): Whether or not to deserialize the data before passing it to the handler.
+            serialization (str, optional): If deserializing, the method of serialization to use; defaults to msgpack.
         """
         if not callable(handler):
             raise TypeError("Passed in handler is not a function!")
         if name in self.reserved_commands and name in self.handler_map:
             raise ValueError(f"'{name}' is a reserved command name dedicated to {name} commands, choose another name")
-        self.handler_map[name] = {"handler": handler, "deserialize": deserialize}
+        self.handler_map[name] = {"handler": handler, "deserialize": deserialize, "serialization": serialization}
```

I don't think we're currently error-checking this. We should identify and raise if an invalid serialization method is passed.